tests/test_cpu.py

```diff
@@ -12,16 +12,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import inspect
+import os
+import sys
 import unittest
+from pathlib import Path
+from unittest.mock import patch
 
+import accelerate
 from accelerate import debug_launcher
+from accelerate.commands.launch import _validate_launch_command, launch_command_parser
 from accelerate.test_utils import require_cpu, test_ops, test_script
+from accelerate.utils.launch import prepare_simple_launcher_cmd_env
 
 
 @require_cpu
 class MultiCPUTester(unittest.TestCase):
+    mod_file = inspect.getfile(accelerate.test_utils)
+    test_file_path = os.path.sep.join(mod_file.split(os.path.sep)[:-1] + ["scripts", "test_cli.py"])
+    test_config_path = Path("tests/test_configs")
+
     def test_cpu(self):
         debug_launcher(test_script.main)
 
     def test_ops(self):
         debug_launcher(test_ops.main)
+
+    def test_mpi_multicpu_config_cmd(self):
```

Comment:
This checks the command, not the actual run of it which is what `test_cpu` is meant to do. Please include a decorator and actually run the test script please here, or move this test to `test_cli.py`